### PR TITLE
Add Dockerfile to support building docker images for Shaderc.

### PR DESCRIPTION
**Pull request by antiagainst**

### CODE REVIEW

**Comment by dneto0** [comment]
+1

**Comment by antiagainst** [comment]
This would be useful for those only interested in running Shaderc out-of-box.

It would be more useful if we can ship official docker images at https://hub.docker.com/r/google, but I'm not sure how to request a slot there. So I've created a image under my personal account: https://hub.docker.com/r/antiagainst/shaderc.

File: README.md [hunk 1]
```diff
@@ -126,6 +126,35 @@ Optionally, the following tools may be installed on any OS:
      `asciidoctor` for syntax highlighting.
  - [`nosetests`](https://nose.readthedocs.org): for testing the Python code.
 
+### Building and running Shderc using Docker
+
+Please make sure you have Docker engine
```
Comment (dekimir) [inline README.md]: "the Docker engine"

File: README.md [hunk 2]
```diff
@@ -126,6 +126,35 @@ Optionally, the following tools may be installed on any OS:
...
+
+Assume `<IMAGE-NAME>` is `shaderc/shaderc` from now on.
+
+To invoke a tool from the above created image in a Docker container:
+
+```bash
+docker run shaderc/shaderc glslc --version
+```
+
+Alternatively, you can mount to the container a host directory (e.g.,
```
Comment (dekimir) [inline README.md]: Consider "mount a host directory [...] containing".

File: README.md (hunk 1 above)
Comment (antiagainst) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (antiagainst) [inline README.md]: Done.

**Comment by dekimir** [comment]
Rebased and merged via d8cad21.

**Pull request closed without merging**